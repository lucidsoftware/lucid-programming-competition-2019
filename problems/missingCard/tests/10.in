20 5
9 17 1 8 13 6 15 3 19 4 11 14 18 7 5 2 16 12 20
11 2 9 16 17 1 8 10 5 4 19 3 13 7 15 14 12 20 18
16 7 14 15 9 11 12 18 2 8 4 10 1 3 17 13 5 19 6
12 8 5 20 16 7 15 17 3 18 11 6 19 1 13 2 10 4 14
7 1 8 15 5 16 14 18 9 2 11 13 6 3 12 10 20 19 17