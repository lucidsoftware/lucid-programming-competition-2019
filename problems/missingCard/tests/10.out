10
6
20
9
4